ConfigurationSystem/private/ServiceInterface.py

```diff
@@ -109,7 +114,96 @@ def __checkSlavesStatus(self, forceWriteConfiguration=False):
                                                 ", ".join(self.dAliveSlaveServers.keys())))
       self.__generateNewVersion()
 
-  def updateConfiguration(self, sBuffer, commiter="", updateVersionOption=False):
+  @staticmethod
+  def __forceServiceUpdate(url, fromMaster):
+    """
+    Force updating configuration on a given service
+
+    :param str url: service URL
+    :param bool fromMaster: flag to force updating from the master CS
+    :return: S_OK/S_ERROR
+    """
+    gLogger.info('Updating service configuration on', url)
+    if url.startswith('dip'):
+      rpc = RPCClient(url)
+      result = rpc.refreshConfiguration(fromMaster)
```

Comment:
I am slightly confused... why do you need an RPC call here?

Reply:
Not sure what is confusing here, we call an external service refreshConfiguration() interface method